features/getting_tasks.feature

```diff
@@ -0,0 +1,157 @@
+Feature: Getting a list of tasks
+
+  Scenario: Three Tasks exist in the Data Store and a get request returns them successfully
+
+    Given I use a service auth token "validServiceAuthToken"
+    And zebedee recognises the service auth token as valid
+    And I have generated a job in the Job Store
+    And I call POST /jobs/{id}/tasks using the generated id
```

Comment:
I would not use other endpoints to setup the data - if they are broke then this test will fail, alluding to Get /tasks not working.

You will want to generate the tasks like you've done for the job on line 7

Reply:
I did try doing it like that but in fact line 7 is using the Get /jobs endpoint - it's just that it's doing it within the jobs_steps rather than in the feature file. The thing that prevented me from doing it that way was that the body (to pass into each step) can only be specified in the feature file (as far as I could see).